cdk/kittyhawk/README.md

```diff
@@ -3,54 +3,114 @@
+Inside the `cdk` folder, initialize a `cdk8s` typescript project. You can do that by following the official cdk8s instructions [here](https://cdk8s.io/docs/latest/getting-started/#new-project). 
 
-You can now run `yarn run build` to try to synthesize your configuration. The generated YAML will be saved to a file located at `k8s/dist/kittyhawk.k8s.yaml`. You should not commit this file. On every push, CircleCI will automatically generate the YAML configuration. On every push to the master branch, CircleCI will also automatically deploy the generated YAML to production.
+## 2. Add Kittyhawk
+Import the kittyhawk library from `yarn` or `npm` and start writing your deployment configuration in Typescript.
 
+At the very top level, add to your `main.ts` file in the following format using the `PennLabsChart`. 
 
-## Examples
+```
+export class MyChart extends PennLabsChart {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+  }
+}
+```
 
-This is a sample configuration deploying a Django Application and a React Application for Penn Clubs:
+Within your chart, you can add various applications, cronjobs, and other configurations supported by kittyhawk, including the following custom constructs:
+- `ReactApplication`
+- `DjangoApplication`
+- `CronJob`
+- `Redis`
 
-```
-import { DjangoApplication, ReactApplication, synth } from '@pennlabs/kittyhawk';
-import { Construct } from 'constructs'; 
-
-export function buildChart(scope: Construct) {
-
-  /** This creates a Django Application named django-asgi **/
-  new DjangoApplication(scope, 'django-asgi', {
-    image: 'pennlabs/penn-clubs-backend',
-    secret: 'penn-clubs',
-    cmd: ['/usr/local/bin/asgi-run'],
-    replicas: 2,
-    domain: 'pennclubs.com',
-    ingressPaths: ['/api/ws'],
-    djangoSettingsModule: 'pennclubs.settings.production',
-    extraEnv: [
-      { name: 'REDIS_HOST', value: 'penn-clubs-redis' }],
-  })
-
-  /** This creates a React Application named react **/
-  new ReactApplication(scope, 'react', {
-    image: 'pennlabs/penn-clubs-frontend',
-    replicas: 2,
-    domain: 'pennclubs.com',
-    ingressPaths: ['/'],
-    portEnv: '80',
-  })
+For more advanced custom constructs, please refer to our [documentation](TODO) (TODO: add link).
 
-}
+### Cron-time-generator
+In many of our Penn Labs products, we use cronjobs to simplify our product tasks (e.g. loading courses, calculating office hours wait time).  
 
-// Synthesizes the chart
-synth(buildChart);
+`cron-time-generator` ([npm link](https://www.npmjs.com/package/cron-time-generator)) is a package that allows for intuitive generation of cron job time expressions. 
 
-```
+It is not required to use this, but it is a developer dependency for kittyhawk and better practice: instead of specifying the time of a cronjob that runs every 5 minutes as `*/5 * * * *`, you can say `cronTime.every(5).minutes()`. 
```

Comment:
I think this should just be a regular dependency. dev dependencies are generally used for linting/other dev stuff whereas the cron package is helpful for the end user